sdk/storage/Azure.Storage.Blobs/tests/PartitionedUploaderTests.cs

```diff
@@ -245,7 +245,41 @@ public async Task AsyncUploadsAsSingleBlockIfUnderLimit()
             Assert.AreEqual(0, testPool.TotalRents);
         }
 
-        private async Task<Response<BlobContentInfo>> InvokeUploadAsync(PartitionedUploader uploader, TestStream content)
+        [Test]
+        [Explicit]
```

Comment:
What does `Explicit` do ?

Reply:
Explicit means that the test must be opted into in some form. According to the nunit website: `The test or fixture will be run if it is selected in the gui, if its name is specified on the console runner command line as the fixture to run, or if it is included by use of a Category filter.`. Note that selecting an enclosing menu in the VS test explorer (e.g. selecting PartitionedUploaderTests or Azure.Storage.Blobs.Test) counts as "selected in the gui," but stating a similar scope on the console does *not* count as "name is specified on the console runner command line as the fixture to run." This means we as devs will likely run the test in our local development cycle but CI will not run this test.

Reply:
Is there a reason not to run the test by default?

Reply:
It uses >4GB of memory and is on the time scale of minutes not milliseconds. Notably, nunit net461 tests run on x86 by default, while netcoreapp tests run on x64 by default. Locally, I have to explicitly specify x64 or the test will have an out of memory exception. I don't believe I have access to alter CI to run x64, and it's probably not a switch to throw without thinking.